### ISSUE TITLE
Could the code use factoring and or simplefication?

### ISSUE THREAD

**Issue by Mario675** [Mario675/Minecraft_Quick_Homes#26]
## There are some functions that could do without another program inside them, that could make the code more readable than just myself (aka author).

> For example:
> - On `HomeWarpCasesSwitch(CaseSwitch, IFSHIFT)`, There is a switch statement inside checking to see if `Option_To_Add_OR_Multiply` is true or not. 
>   - This could make up for code, where if I were to add another feature regarding changing the final `CaseSwitch` amount It could need to find this switch statement. 
>     - Instead, I can put that switch check into a function, and have these checks point to databases like classes. 
```cpp
check_modifier(int type_modifier_value)
{
  switch (type_modifier_value)
  {
    add_ten:
       //here's some cpp that I picked up
      modifier_varied_actions::add_ten();
      break;
    multiply:
      modifier_varied_actions::multiply();
      break;
      
    //etc
  }
}
```

The `if_shift`, or better known as the modifier value, can be determined by a function. Which would have an switch statement in it. That switch statement would reference actions from the classes. 
 - Then, It could have it return an int.
>
> TL;DR
> - You would not have to understand all the code at once to get the main premises of the code.
> - You would not have to modify the code, and look for the detail much. 
> - It would be better for the end user for them to modify the code for their own use, without it going too out of hand.
> - It would make it simple at the beginning, and that you would not need to question the use. Meaning, you can go deep in detail if you wanted, but you didn't have to read it. 

## Where did I get this incentive to clean up my own code?

- Why from [Clean Code - Uncle Bob / Lesson 1](https://youtu.be/7EmboKQH8lM) of course.

**Comment by Mario675** [comment]
Added the Alt + I shortcut, which opens up the current settings being used. 
06ddfb4d7ad10d66c6c9a3a77f0bb00e256ea3d4
0e3fa85a90dd96f02329533260d195354877e76f

**Comment by Mario675** [comment]
Issue has been closed due to the issue being generalized, and fitting all other issues within refactoring. 
Please report specific issues. Thank you.
